### ISSUE TITLE
Consolidate understanding documentation

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#2992]
### Section

https://www.w3.org/TR/epub-a11y-12/#sec-epub-req

### Describe the problem

As part of creating an "understanding epub accessibility 1.2" document to address some of the informative guidance  requested in some of the other issues, I've been looking over the structure of the current accessibility and techniques documents and I think we can do a better job of organizing the information.

But given that we want to move to CR soon, some of what I'd like to address should be done now as it involves removing some existing understanding documentation from the accessibility standard. That gives us more time to refine this new note.

### Describe the fix or new feature you propose

What I've been playing around with is as follows:

- to move all the "understanding this objective" sections from the EPUB objectives to the new document. We only put those in the spec because we didn't have anyplace else at the time. We can link from the objective purpose to the explainers instead.
- merge the sections in the techniques document on the difference between epub:type and roles and not repeating roles in chunked content with the [EPUB type to ARIA role guide](https://www.w3.org/TR/epub-aria-authoring-11/). I'm not sure why we duplicated them in the techniques when we have a guide already dedicated to this (plus one isn't a technique and the other isn't directly linked to a WCAG success criteria).
- move the remaining section on using epub landmarks to the content access section of the techniques as landmarks are another way to meet multiple ways (even if that SC is already covered by many aspects of epub).
- move the sections on bypass blocks and multiple ways from the techniques to the new understanding document because they are providing guidance and not techniques.

That should clean up the standard and techniques and let us focus on getting the new understanding doc done. But if anyone has other ideas, feel free to comment.

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 07 May 2026](https://w3c.github.io/pm-wg/minutes/2026-05-07-a11y.html#8ed3).

<details><summary><i>View the transcript</i></summary>

<h3>Matt has updated the understanding document: Issue #2992</h3>
<p><cite>&lt;AvneeshSingh&gt;</cite> <a href="https://github.com/w3c/epub-specs/issues/2992" rel="noopener noreferrer">w3c/<wbr>epub-specs#2992</a></p>
<p><cite>&lt;AvneeshSingh&gt;</cite> <a href="https://github.com/w3c/epub-specs/pull/2995" rel="noopener noreferrer">w3c/<wbr>epub-specs#2995</a></p>
<p><cite>MattG:</cite> it's about taking out the explanatory for a dedicated document and avoid duplications.<br>
<span>… there's more we can do, but one step after the other.</span><br>
<span>… there's a second PR with all the extracted text + a section on reflow, so people can review if it's comprehensive enough. I'm planning for the understanding document to PR one section at a time so it's easier to review.</span><br>
<span>… the first document is to be included in 3.4 CR.</span></p>
<p><cite>AvneeshSingh:</cite> it's still difficult to review those previews with a screen reader, we should we find an easier way.<br>
<span>… diff is accessible, but it generates a lot of noise like earing plus minus plus minus.</span></p>
<p><cite>GeorgeK:</cite> I may review the whole thing, not diff by diff.</p>
<p><cite>MattG:</cite> there's no text modification, just removed texts.</p>

<hr /></details>
